src/main/webapp/app/modules/canvas/features/features-panel.tsx

```diff
@@ -33,6 +53,11 @@ const FeaturesPanel = (props: IFeaturesPanelProp) => {
               <Text>
                 <span>Features</span>
```

Comment:
please use for heading and put text in json 
https://opensource.adobe.com/spectrum-css/typography.html
for example:

   <span className="spectrum-Heading spectrum-Heading--sizeXXS">
                <Translate contentKey="features.title">Features</Translate>
     </span>

Reply:
Adding text to json, but I did not understand regarding the heading. I made it exactly the same as filter-panel.tsx has been made. Could you elaborate please?

Reply:
heading was just an example.it does not mean u need use the same.look for suitable class in the given link. whether u need a body,heading or details. we need to place text inside spectrum typography as we need the consistent text across application. if spectrum typography was not used inside filter panel text..that is also not correct